src/scripts/dungeons/Dungeon.ts

```diff
@@ -115,34 +150,17 @@ class Dungeon {
         return list.filter(Dungeon.isPokemon);
     }
 
-    get availablePokemon(): PokemonNameType[] {
-        return this.pokemonList.concat(this.bossPokemonList);
-    }
-
     /**
-     * Retreives the weights for all the possible enemies
+     * Gets all possible Pokemon in the dungeon
      */
-    get weightList(): number[] {
-        return this.enemyList.map((enemy) => {
-            if (typeof enemy === 'string') {
-                return 1;
-            } else if (enemy.hasOwnProperty('pokemon')) {
-                return (<DetailedPokemon>enemy).options.weight ?? 1;
-            } else {
-                return (<DungeonTrainer>enemy).options?.weight ?? 1;
-            }
-        });
+    get availablePokemon(): PokemonNameType[] {
+        return this.pokemonList.concat(this.bossPokemonList);
```

Comment:
Should this be called something like `allPokemon`?
as this is a list of all possible Pokémon, not necessarily available (locked by requirements)

Reply:
Updated